varaibles.tf

```diff
@@ -16,6 +16,36 @@ variable "node_instance_type" {
   default     = "t2.small.elasticsearch"
 }
 
+variable "dedicated_master_type" {
+  description = "Instance type of dedicated master node."
+  default     = "t3.small.elasticsearch"
+}
+
+variable "dedicated_master_count" {
+  description = "Amount of dedicated master nodes."
+  default     = "0"
+}
+
+variable "index_slow_log_group_arn" {
+  description = "XXX"
+  default     = ""
+}
+
+variable "search_slow_log_group_arn" {
+  description = "XXX"
+  default     = ""
+}
+
+variable "es_application_log_group_arn" {
+  description = "XXX"
+  default     = ""
+}
+
+variable "audit_log_group_arn" {
+  description = "XXX"
```

Comment:
Let's change XXX to some obvious, but human-readable "Audit Log Group ARN" for `audit_log_group_arn` :)